charts/refinery/ci/rules-values.yaml

```diff
@@ -0,0 +1,17 @@
+resources:
+  limits:
+    cpu: 100m
+    memory: 200M
+  requests:
+    cpu: 100m
+    memory: 200M
+
+rules:
+  RulesVersion: 2
+  Samplers:
+    __default__:
+      DeterministicSampler:
+        SampleRate: 100
```

Comment:
Is 100 the value we want here?

Reply:
This is for a test, proving that we can set a default sampler and it will merge correctly.  I chose a non-default value to show that the value really is changing.